core/src/test/resources/filtered/streamx-mesh.yml

```diff
@@ -13,7 +13,7 @@ processing:
         topic: outboxes/pages
 
 delivery:
-  web-delivery-service:
+  web:
```

Comment:
Similar as done in [[DXP-839] Prefix names of processing and delivery containers](https://github.com/streamx-dev/streamx/pull/125)